.github/workflows/github-actions.yml

```diff
@@ -0,0 +1,30 @@
+name: Build and Test
+on:
+  push:
+    branches: [ "main" ]
+  pull_request:
+    branches: [ "*" ]
+
+  workflow_dispatch:
+
+jobs:
+  docker:
```

Comment:
Why is the whole actions called a docker, even though it does more than that?

Reply:
I rename this , but lets discuss on meeting